Source/WebCore/platform/MIMETypeRegistry.cpp

```diff
@@ -832,4 +833,18 @@ Vector<String> MIMETypeRegistry::allowedFileExtensions(const Vector<String>& mim
     return allowedFileExtensions;
 }
 
+bool MIMETypeRegistry::isJPEGMIMEType(const String& mimeType)
+{
+#if USE(CG)
+    auto destinationUTI = utiFromImageBufferMIMEType(mimeType);
+    if (!destinationUTI)
+        return false;
+    return CFEqual(destinationUTI.get(), jpegUTI());
+#else
+    return mimeType.substring(sizeof "image") == "jpeg"_s;
+#endif
+    ASSERT_NOT_REACHED();
+    return false;
```

Comment:
These two lines are dead code, regardless of which part of the `#if` is built; they should be removed.

Reply:
Fixed.